linclogger/linclogger.py

```diff
@@ -1,8 +1,9 @@
 import logging
 import os
-from cloghandler import ConcurrentRotatingFileHandler
 from logging import WARNING as WARNING_LEVEL
 
+from cloghandler import ConcurrentRotatingFileHandler
+
 __version__ = '0.1.1'
```

Comment:
you want to change version to tag that you are going to create.

Reply:
oh yes good catch